src/charm.py

```diff
@@ -178,7 +212,9 @@ def _on_database_relation_broken(self, event: EventBase) -> None:
         Args:
             event: Juju event
         """
-        self.unit.status = BlockedStatus("Waiting for database relation")
+        for relation in ["database", "auth_database"]:
```

Comment:
can use `DATABASE_RELATION_NAME` and create one constant for `"auth_database"`

Reply:
Done